### ISSUE TITLE
[Bug] Solution files containing spaces are not correctly escaped

### ISSUE THREAD

**Issue by broodroosterdev** [resharper-community/resharper-vscode#2]
I have a project that has a solution file containing spaces. This results in the extension being unable to inspect the code.

Example log:
```
Inspect Code command is running for 'd:\test-project\Test Project.sln'...
jb inspectcode --output=d:\test-project\inspectcode.xml d:\test-project\Test Project.sln
JetBrains Inspect Code 2023.3.4
Running on x64 OS in x64 architecture, .NET 8.0.1 under Microsoft Windows 10.0.19045
Specify only one solution file
Usage: inspectcode [options] [solution or project file]
Show help: inspectcode --help
```

**Comment by tony-xia** [comment]
Good catch! will fix it in the next release.

**Comment by tony-xia** [comment]
fixed in 0.9.13
